manual.md

```diff
@@ -7,10 +7,10 @@
 ---
 ## Table of Contents
 + [0. Licenses](#section0)
-+ [1. Getting started](#section1)
++ [1. Getting Started](#section1)
```

Comment:
If you change the style of the title, probably better to do it in a separate PR because this change brings an inconsistency to the manual after the merge.